java-util/src/main/java/io/druid/java/util/common/parsers/TimestampParser.java

```diff
@@ -47,13 +50,21 @@
         public DateTime apply(String input)
         {
           Preconditions.checkArgument(input != null && !input.isEmpty(), "null timestamp");
+          String[] inputArray = input.split(" ");
+          String timeZone = "UTC";
```

Comment:
Could avoid going "string" path in default (UTC) case? Using `DateTimes.utc(instant)`